src/Microsoft.Fx.Portability/CompressedHttpClient.cs

```diff
@@ -3,6 +3,7 @@
 
 using Microsoft.Fx.Portability.Reporting.ObjectModel;
 using Microsoft.Fx.Portability.Resources;
+using Newtonsoft.Json;
```

Comment:
Why is this needed? I may be overlooking it, but I don't see anything from the namespace used in the new code below.